### DISCUSSION TITLE
How to use with indent-blankline.nvim ?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [folke/noice.nvim#270] [Q&A] [answered]
I'm using Noice with [`indent-blankline.nvim`](https://github.com/lukas-reineke/indent-blankline.nvim) and indent lines always appear in Noice messages. It's kinda annoying.
[Screencast from 12-18-2022 02:00:53 PM.webm](https://user-images.githubusercontent.com/87268103/208293508-cfd14e15-af0b-4b98-a83a-f89d5acd5784.webm)
I didn't have this problem with `fidget.nvim`. How should I fix it?

**Comment by folke** [comment, marked as answer]
Did you actually read the indent blankline docs? https://github.com/lukas-reineke/indent-blankline.nvim/blob/c4c203c3e8a595bc333abaf168fcb10c13ed5fb7/doc/indent_blankline.txt#L393

filetype is `noice`

**Reply by amirhhashemi** [reply]
Thank you. I didn't know that the filetype is `noice`. Actually I didn't know that those small messages could have a filetype.

**Reply by folke** [reply]
every buffer can have a filetype and anything that you see in Neovim is displayed in a buffer somewhere (except for statuslines etc)
